### PR TITLE
[WIP] edit the date format string

### PR DESCRIPTION
**Pull request by antis81**
If I did everything right, this fixes issue #2189.

The edited date format string is saved in the qTox settings. It is overwritten by selecting one of the default date format strings.

Who likes to review?

### CODE REVIEW

**Comment by antis81** [comment]
Apparently I did not do everything right yet… Does not crash though, phew. :sweat_smile:

Anyway, an entered text format is simply ignored…

**Comment by agilob** [comment]
Could you do the same for timestamps to make it look more consitent?

**Comment by antis81** [comment]
@agilob This is not working as is. Let me think about it a little more…

**Comment by zetok** [comment]
um, if it's WIP, mind prefixing PR title with `[WIP]` ?

**Comment by agilob** [comment]
@antis81 do you have time do finish that branch?

**Comment by agilob** [comment]
bump?

**Comment by antis81** [comment]
@agilob Don't think I can finish it soon -> simply not enough time and my focus is on audio stuff…

If anyone can, feel free to work on this one or come up with a new PR…

**Comment by agilob** [comment]
OK, than take care of audio, I'll fork this

**Comment by agilob** [comment]
Moving this to a new branch.

File: src/widget/form/settings/generalform.cpp [hunk 1]
```diff
@@ -41,8 +41,6 @@ static QStringList locales = {"bg", "cs", "de", "en", "es", "fr", "hr", "hu", "i
 static QStringList langs = {"Български", "Čeština", "Deutsch", "English", "Español", "Français", "Hrvatski", "Magyar", "Italiano", "Lietuvių", "Nederlands", "Norsk Bokmål", "Polski", "Português", "Русский", "Slovenščina", "Suomi", "Svenska", "Українська", "简体中文"};
 
 static QStringList timeFormats = {"hh:mm AP", "hh:mm", "hh:mm:ss AP", "hh:mm:ss"};
-// http://doc.qt.io/qt-4.8/qdate.html#fromString
-static QStringList dateFormats = {"yyyy-MM-dd", "dd-MM-yyyy", "d-MM-yyyy", "dddd d-MM-yyyy", "dddd d-MM", "dddd dd MMMM"};
```
Comment (agilob) [inline src/widget/form/settings/generalform.cpp:45]: I don't like this change, it removes date format which I use so, I'll add it back.

**Pull request closed without merging**